bindings/Python/py11glue.cpp

```diff
@@ -366,6 +366,10 @@ PYBIND11_MODULE(adios2, m)
         .def("Open", (adios2::py11::Engine (adios2::py11::IO::*)(
                          const std::string &, const int)) &
                          adios2::py11::IO::Open)
+        .def("Open",
```

Comment:
this need to be tested from a Python program, so it can run under CI. Feel free to add tests under testing/adios2/bindings/python calling this function. Only MPI tests are needed.

Reply:
Done (see below).